src/node/share_manager.h

```diff
@@ -18,41 +18,30 @@
 
 namespace ccf
 {
-  class LedgerSecretWrappingKey
+  class SharedLedgerSecretWrappingKey
   {
   private:
     static constexpr auto KZ_KEY_SIZE = ccf::crypto::GCM_DEFAULT_KEY_SIZE;
     bool has_wrapped = false;
     size_t num_shares;
     size_t recovery_threshold;
     std::vector<uint8_t> data; // Referred to as "kz" in TR
```

Comment:
In wrap():84, let's create (and scrub) this on the fly, and only keep secret as a member attribute